docs/daily-tests.html

```diff
@@ -0,0 +1,356 @@
+
+    }
+
+    function destroyCharts() {
+        if (throughputChart !== undefined) {
+            throughputChart.destroy();
+        }
+        if (averageChart !== undefined) {
+            averageChart.destroy();
+        }
+        if (stdDevChart !== undefined) {
+            stdDevChart.destroy();
+        }
+        if (percentileChart !== undefined) {
+            percentileChart.destroy();
+        }
+        if (errorChart !== undefined) {
+            errorChart.destroy();
+        }
+    }
+
+    function extractData(stats, users, payload, attribute) {
+        let coordinates = [];
+        stats.filter(function (row) {
+            return row['Users'] === users && row['Payload'] === payload;
+        }).map(function (value) {
+                let date = new Date(value['Date'] * 1000);
+                if (fromDate.getTime() < date.getTime() && date.getTime() < toDate.getTime()) {
+                    coordinates.push({"x": date.toLocaleString(), "y": value[attribute].replace("%", ""), "version": value['Version']});
+                }
+            }
+        );
+        coordinates = sort_by_key(coordinates, 'x');
+        return coordinates;
+    }
+
+    function sort_by_key(array, key) {
+        return array.sort(function (a, b) {
+            let x = new Date(a[key]).getTime();
+            let y = new Date(b[key]).getTime();
+            return ((x < y) ? -1 : ((x > y) ? 1 : 0));
+        });
+    }
+
+    function getCSVData(stats, attribute, chartTypes) {
+        let dataset = [];
+        if (chartTypes !== undefined) {
+            chartTypes.forEach(function (type) {
+                let res = type.split("_");
+                let users = res[0];
+                let payload = res[1];
+                let r = Math.floor(Math.random() * 255);
+                let g = Math.floor(Math.random() * 255);
+                let b = Math.floor(Math.random() * 255);
+                let color = 'rgb(' + r + ',' + g + ',' + b + ')';
+                let data = {
+                    label: users + " users " + payload + " payload (ms)",
+                    showLine: true,
+                    backgroundColor: color,
+                    borderColor: color,
+                    data: extractData(stats, users, payload, attribute)
+                }
+                dataset.push(data);
+            })
+        }
+        return {
+            datasets: dataset
+        };
+    }
+
+    function drawCharts(url, chartTypes) {
+        destroyCharts();
+        d3.csv(url).then(function (stats) {
+            //Throughput Chart
+            const throughput_dataset = getCSVData(stats, 'Throughput', chartTypes)
+            throughputChart = drawChart(throughput_dataset, 'throughputCanvas');
+
+            // Average Chart
+            const average_dataset = getCSVData(stats, 'Average', chartTypes);
+            averageChart = drawChart(average_dataset, 'averageCanvas');
+
+            //Standard Deviation of Response Time Chart
```

Comment:
```suggestion
            //Standard Deviation of Response Time Chart
```
Better to remove title case in code comments.